### ISSUE TITLE
ERESOLVE unable to resolve dependency tree

### ISSUE THREAD

**Issue by waveiron** [embarq/safe-pipe#6]
```text
npm install safe-pipe

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: angular@0.0.0
npm ERR! Found: @angular/common@11.2.3
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"^11.2.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"~9.1.1" from safe-pipe@1.0.3
npm ERR! node_modules/safe-pipe
npm ERR!   safe-pipe@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by embarq** [comment]
Hi @waveiron!
Thanks for the report.
Could you check if it works for you if you switch to [v2.0.2](https://www.npmjs.com/package/safe-pipe/v/2.0.2)?

**Comment by alexdinescu** [comment]
hello,
```
npm ERR! Found: @angular/common@13.3.6
npm ERR! node_modules/@angular/common
npm ERR!   dev @angular/common@"~13.3.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"~13.1.1" from safe-pipe@2.0.2
npm ERR! node_modules/safe-pipe
npm ERR!   safe-pipe@"^2.0.2" from the root project
```
with the latest 2.0.2 version angular 13.3.x is still 'incompatible' as it has a peer dep fixed on 13.1.x
can you bump it ?

**Comment by kumarmanishc** [comment]
@embarq I'm also having same issue with angular 14.


"@angular/core": "^14.2.0",

**Comment by xpusostomos** [comment]
So safe-pipe is a dead project is it? Nobody is fixing even the most rudimentary issues?

**Comment by embarq** [comment]
Fixed in #24
